src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -117,6 +145,27 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Returns the fallback template for the given slug.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param WP_REST_Request $request The request instance.
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function get_template_fallback( $request ) {
+		if ( empty( $request['slug'] ) ) {
+			return new WP_Error(
+				'rest_invalid_param',
+				__( 'Invalid slug.' ),
+				array( 'status' => 400 )
+			);
+		}
+		$hierarchy         = get_template_hierarchy( $request['slug'], $request['is_custom'], $request['template_prefix'] );
+		$fallback_template = resolve_block_template( $request['slug'], $hierarchy, '' );
+		return rest_ensure_response( $fallback_template );
```

Comment:
```suggestion
		$response = $this->prepare_item_for_response( $template, $request );
		return rest_ensure_response( $response );
```

Please ensure that response is run through `prepare_item_for_response`, so the response is formatted correctly.

Reply:
I'll update but this change will also need a GB PR(therefore a packages update) to retrieve the content from `content.raw`.

Reply:
Make the change and backport it. I can approve the change there as well. Change the properties / shape of response can / will cause problems down the road.

Reply:
GB PR is here: https://github.com/WordPress/gutenberg/pull/44299 --cc @ockham